Allow users to request a new 2fa security code (#573)
UKGovernmentBEIS/beis-opss-psd: CHANGELOG.md

# Changelog
All notable changes to this project will be documented in this file.

## 2020-05-06

> - Allow to request a new secondary authentication code when secondary authentication is required and code was not received via sms.

Reviewer: Suggested rewording:

```suggestion
- Allow users to request a new security code when the code was not received via SMS.
```

Author: Thanks for this!
